tensorboard/defs/vulcanize.bzl

```diff
@@ -18,7 +18,7 @@ load("//tensorboard/defs:web.bzl", "web_aspect")
 
 def _tensorboard_html_binary(ctx):
   deps = unfurl(ctx.attr.deps, provider="webfiles")
-  manifests = depset(order="topological")
+  manifests = depset(order="postorder")
```

Comment:
I thought I fixed this. Do you need to sync to head? I've also been fixing this to be `preorder` (i.e. somewhat reverse topological) so webfiles get overridden in the correct order. If that's not the case, then maybe Vulcanize.java needs to be updated?